### PR TITLE
fix: GH issue 8669 NVD3 tooltip overflow

### PR DESCRIPTION
**Pull request by rusackas**
🐛 Bug Fix

This addresses Superset issue 8669:
https://github.com/apache/incubator-superset/issues/8669

It also makes the headers bold, which hopefully looks better to folks.

Before:
image

After:
image

### CODE REVIEW

File: packages/superset-ui-legacy-preset-chart-nvd3/src/NVD3Vis.css [hunk 1]
```diff
@@ -121,3 +121,13 @@
...
+
+/* START tooltip styles - these prevent long column names from overflowing the tooltip */
+.nvtooltip tbody tr:first-child td{
+  white-space: nowrap;
+  font-weight: bold;
+}
+.nvtooltip tbody tr:not(:first-child) td:nth-child(2){
+  word-break: break-word;
+}
+/* END tooltip styles */
```
Comment (etr2460) [inline packages/superset-ui-legacy-preset-chart-nvd3/src/NVD3Vis.css]: nit: add a new line

**Review by etr2460 — APPROVED** [review]
lgtm with one nit

File: packages/superset-ui-legacy-preset-chart-nvd3/src/NVD3Vis.css (hunk 1 above)
Comment (rusackas) [reply packages/superset-ui-legacy-preset-chart-nvd3/src/NVD3Vis.css]: Committed. I wish people didn't object to using Husky to add pre-commit hooks running Prettier

Comment (williaster) [reply packages/superset-ui-legacy-preset-chart-nvd3/src/NVD3Vis.css]: who objects to husky pre-commit hooks? I support that 👍

Comment (williaster) [reply packages/superset-ui-legacy-preset-chart-nvd3/src/NVD3Vis.css]: on second thought, I'm not so sure. we've had these in other repos and  the major downside of pre-commit hooks is that they can take longer and longer to run, especially as the repo grows. 

I think you can setup your `ide` to auto-fix eslint (which calls prettier) changes on save.

**Pull request merged**